pwa/src/components/common/layout.tsx

```diff
@@ -22,9 +23,26 @@ import favicon from "../../images/conduction_logo_blauw.svg";
 export default function Layout({ children, title = "", subtext = "" }) {
   const [API, setAPI] = React.useState<APIService>(null)
 
+  const parseJwt = (token) => {
+    try {
+      return JSON.parse(atob(token.split(".")[1]));
```

Comment:
The use of `atob` is advised against. My IDE gives the following reason: 

```
This function is only provided for compatibility with legacy web platform APIs and should never be used in new code, because they use strings to represent binary data and predate the introduction of typed arrays in JavaScript. For code running using Node.js APIs, converting between base64-encoded strings and binary data should be performed using Buffer.from(str, 'base64') andbuf.toString('base64').
```

Might be worth to check if we can use another alternative.